packages/vscode/src/plugins/prisma-postgres-manager/PrismaPostgresRepository.ts

```diff
@@ -230,17 +230,19 @@ export class PrismaPostgresApiRepository implements PrismaPostgresRepository {
 
     let createdDatabase: RemoteDatabase | undefined
     let connectionString: string | undefined
-    const createdDatabaseData = response.data.databases[0]
-    if (createdDatabaseData) {
-      createdDatabase = {
-        type: 'remoteDatabase' as const,
-        id: createdDatabaseData.id,
-        name: createdDatabaseData.name,
-        region: null,
-        projectId: newProject.id,
-        workspaceId,
+    if ('databases' in response.data) {
+      const createdDatabaseData = response.data.databases[0]
```

Comment:
You could change this to

```suggestion
      const createdDatabaseData = response.data.databases?.[0]
```

and get rid of the outer if statement you added

Reply:
Unfortunately not type wise :/
<img width="787" alt="Screenshot 2025-05-22 at 07 34 40" src="https://github.com/user-attachments/assets/31f0c41a-d29e-461c-b1be-951b89399d8e" />